Add a native PulseAudio audio module.
This patch was originally developed by Matthias Clasen and submitted to:
http://www.redhat.com/archives/fedora-extras-commits/2009-September/msg09311.html
This patch depends on pulseaudiosupport.diff patch in speech-tools.

--- a/src/arch/festival/festival.cc
+++ b/src/arch/festival/festival.cc
@@ -344,6 +344,8 @@
 	proclaim_module("nas");
     if (esd_supported)
 	proclaim_module("esd");
+    if (pulse_supported)
+	proclaim_module("pulseaudio");
     if (sun16_supported)
 	proclaim_module("sun16audio");
     if (freebsd16_supported)
